### PR TITLE
docs: add javascript code comments

### PR DESCRIPTION
**Pull request by mauriciomutte**
I've added rules about code comments in Javascript.

## What was added?
6.5 Code Comments

### CODE REVIEW

**Review by deverebor — APPROVED** [review]

**Comment by rogigs** [comment]
I agree that we should avoid writing comments in the code as much as possible, as the code should be self-explanatory. However, I believe that code snippets that are complex by themselves should have some kind of comment to help, and for these cases I believe that the best thing to do is to use JsDoc. Could we put a note on this, what do you think @mauriciomutte ?

File: README.md [hunk 1]
```diff
@@ -690,6 +691,28 @@ const sendMessage = (message, channel) => {
 }
 ```
 
+### 6.5 Code Comments
+
+Avoid writing comments to explain the code.
```
[the remark this answers is not in the document]
Comment (felipefialho) [reply README.md:696]: Maybe we could add some examples that make sense add explanation

[the remark this answers is not in the document]
Comment (rogigs) [reply README.md:696]: I found a explanation that I thinked interessantig. We could put like this:

```suggestion
Avoid writing comments to explain the code.

...

Ps. If you need to use comments, use JsDocs

```js
/**
 *  Capture file from a given location
 *  @param {string} name - Name of file
 *  @returns {string} pathname - File path
 */
const getFileByName = ({ name, pathname }) => {
  // do something
};
```

What do you thinl ? @felipefialho @mauriciomutte

[the remark this answers is not in the document]
Comment (mauriciomutte) [reply README.md]: I think JsDocs might not be good in most cases. It's what we were discussing in thread for this topic. I just add cases for good comments. What do you think?

**Review by RodrigoRVSN — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (felipefialho) [reply README.md]: I agree @mauriciomutte

[the remark this answers is not in the document]
Comment (rogigs) [reply README.md]: I understand your point and I agree with your case.

**Pull request merged**